app/src/main/java/org/fossasia/openevent/general/event/Event.kt

```diff
@@ -70,4 +71,12 @@ data class Event(
     @ColumnInfo(index = true)
     @Relationship("event-topic")
     var eventTopic: EventTopic? = null
-)
+) {
+    fun getStrippedDescription(): String? {
+        return this.description?.stripHtml()
+    }
+
+    fun getStrippedOrganizerDescription(): String? {
+        return this.organizerDescription?.stripHtml()
+    }
```

Comment:
When you have created an extension function, then no need of these

Reply:
you said to make a new getter in data class how else I could have done it???

Reply:
Now that you have created an extension function, you can just write `event.description.getStrippedHtml()`

Reply:
k..